app/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepository.scala

```diff
@@ -204,8 +208,13 @@ class CaseMongoRepository @Inject()(mongoDbProvider: MongoDbProvider, mapper: Se
       .collect[List](Int.MaxValue, reactivemongo.api.Cursor.FailOnError())
       .map {
         _.map { obj: JsObject =>
+
           ReportResult(
-            Option(obj.value("_id")).filter(_.isInstanceOf[JsString]).map(_.as[JsString].value),
+            obj.value("_id") match {
+              case JsObject(map) => Some(s"${map.getOrElse("queueId",JsString("")).as[JsString].value}-${map.getOrElse("type",JsString("")).as[JsString].value}")
```

Comment:
Think we should pull this out rather than using String interpolation

Reply:
This is going to be superseded by the generic grouping solution anyway.